flaskapp/view_functions/ivrflow.py

```diff
@@ -269,7 +279,7 @@ def save_feedback():
         sheet.append_row(new_row)
         send_mail("FEEDBACK", phone=phone, feedback=msg)
     except:
-        return('-1')
+        return ('-1')
     return ('0')
```

Comment:
Why not to return just True or False... `('0')` is too complex, ` return '0' ` is ok.

Reply:
done